sdk/keyvault/azure-security-keyvault-certificates/src/certificate_serializers.cpp

```diff
@@ -381,3 +381,47 @@ std::string CertificateCreateParametersSerializer::Serialize(
 
   return parameter.dump();
 }
+
+std::string CertificateContactsSerializer::Serialize(
+    std::vector<CertificateContact> const& contacts)
+{
+  json response;
+
+  for (auto contact : contacts)
+  {
+    json contactJson;
+
+    JsonOptional::SetFromNullable(contact.EmailAddress, contactJson, EmailPropertyName);
+    JsonOptional::SetFromNullable(contact.Name, contactJson, NamePropertyName);
+    JsonOptional::SetFromNullable(contact.Phone, contactJson, PhonePropertyName);
+
+    response[ContactsPropertyName].emplace_back(contactJson);
```

Comment:
how would the json look for a contacts array of size 3 with all contacts with null values?

would it be like? : 
```json
{ "contactsPropertyName": [ { }, { } , { } ]  }
```

What would the server do?  

I wonder if we would want to call `emplace_back` only if the contactJson has at least one property

Reply:
done , email is required